### PR TITLE
Reformat distributions table

### PR DESCRIPTION
**Pull request by terrytangyuan**
This ensures everything is centered correctly and nicely formatted in editor.

### CODE REVIEW

**Comment by ashwinb** [comment]
But this makes it really hard to maintain this table. I think we should use the facilities of markdown to render this table instead of ourselves centering it like this honestly.

**Comment by terrytangyuan** [comment]
Any IDEs with markdown formatter should be able to format this for us automatically going forward. Previously this was not possible because we had a few missing “|”s in the table.

**Comment by terrytangyuan** [comment]
Just to clarify, before this PR, I was not able to use any markdown formatter to automatically format everything. After this PR, developers can use a formatter, including a formatter used within IDEs like PyCharm and VS Code. They no longer need to format/center manually.

**Review by ashwinb — APPROVED** [review]
fair enough!

File: README.md [hunk 1]
```diff
@@ -84,26 +84,26 @@ Additionally, we have designed every element of the Stack such that APIs as well
 |                                         Fireworks                                          |         Hosted         | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |
```
Comment (aidando73) [inline README.md:84]: @terrytangyuan What formatter are you using?

I'm using the default Vscode formatting and getting different results:

image

Comment (terrytangyuan) [reply README.md:84]: Hmm interesting. I used the default markdown plugin in PyCharm.

**Pull request merged**